setup-elytron-guides/setup_elytron_linux.adoc

```diff
@@ -0,0 +1,148 @@
+---
+layout: setup-elytron
+title: Setting up WildFly Elytron on Linux & WSL
+permalink: /guides/setup-elytron-linux/
+os: linux
+---
+:imagesdir: ../../assets/images/setup-elytron/
+
+:toc: macro
+:toc-title:
+
+This page will guide you through the steps needed to setup a development environment for WildFly Elytron on Linux, or the https://docs.microsoft.com/en-us/windows/wsl/about[Windows Subsystem for Linux (WSL)]. For WSL developers, all commands should be entered into the WSL terminal _(i.e. not Command Prompt, PowerShell, or Git Bash)_.
+
+toc::[]
+
+== Installing Java
+
+. First, let's check if you already have Java installed.
+    .. Open a terminal, and run `java --version`.
+    .. If the output shows a version number, check to make sure it is Java 11 (e.g., "11.0.15.1"). If so, Java 11 is already installed and you can jump to the *Installing Maven* section. If the output is not Java 11, or if the output is `command not found: java` or a similar message indicating you do not have Java installed, then continue with the following steps.
+. It is easiest to install Java on Linux using the preferred package manager of your Linux distribution. Here are some examples of how to install Java on common Linux distributions:
+    ** Ubuntu: `sudo apt-get install openjdk-11-jdk`
+    ** Fedora: `sudo dnf install java-11-openjdk`
+    ** Arch: `sudo pacman -S java-11-openjdk`
+. Next, configure the default JVM version for your system:
+    ** Ubuntu: `sudo update-alternatives --config java`
+    ** Fedora: `sudo alternatives --config java`
+    ** Arch: `archlinux-java set java-11-openjdk`
+. Configure JAVA_HOME and PATH environment variables for Java:
+    .. Open the configuration file for your shell, which you can identify by typing `echo $0`. If the response is `bash`, open the file _~/.bashrc_, and if it is `sh`, edit _~/.profile_. You can look up config files for other shells by searching the name of the shell, followed by "config file" (ex. "zsh config file").
```

Comment:
@cam-rod  I am guessing that the Windows WSL which is mentioned in the title and the beginning, does not have  ~/.bashrc or ~/.profile? Shouldn't they follow windows guide for confiuring JAVA_HOME and PATH?